src/SPECS/hootenanny.spec.in

```diff
@@ -337,11 +318,11 @@ EOT
 
     # Configuring firewall
     if ! sudo iptables --list-rules | grep -i --quiet 'dport 80'; then
-        sudo iptables -A INPUT -p tcp -m state --state NEW -m tcp --dport 80 -j ACCEPT
-        sudo iptables -A INPUT -p tcp -m state --state NEW -m tcp --dport 8080 -j ACCEPT
-        sudo iptables -A INPUT -p tcp -m state --state NEW -m tcp --dport 8000 -j ACCEPT
-        sudo iptables -A PREROUTING -t nat -p tcp --dport 80 -j REDIRECT --to-ports 8080
-        sudo iptables -A OUTPUT -t nat -s 0/0 -d 127/8 -p tcp --dport 80 -j REDIRECT --to-ports 8080
+        sudo iptables -I -A INPUT -p tcp -m state --state NEW -m tcp --dport 80 -j ACCEPT
+        sudo iptables -I -A INPUT -p tcp -m state --state NEW -m tcp --dport 8080 -j ACCEPT
+        sudo iptables -I -A INPUT -p tcp -m state --state NEW -m tcp --dport 8000 -j ACCEPT
+        sudo iptables -I -A PREROUTING -t nat -p tcp --dport 80 -j REDIRECT --to-ports 8080
+        sudo iptables -I -A OUTPUT -t nat -s 0/0 -d 127/8 -p tcp --dport 80 -j REDIRECT --to-ports 8080
```

Comment:
`-I -A` tells you that I didn't read the man page for iptables ;-)

I'll remove the append flag.